### PR TITLE
Fix rpcClient.Address() race condition

### PR DESCRIPTION
**Pull request by bitwiseguy**
Fix `rpcClient.Address()` race condition that caused a CI failure when changes were merged into `main`: https://github.com/statechannels/go-nitro/actions/runs/6012869180/job/16309076480

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -178,7 +178,7 @@ func main() {
 
 			logging.SetupDefaultLogger(os.Stdout, slog.LevelDebug)
 
-			node, _, _, _, err := node.InitializeNode(pkString, chainOpts, useDurableStore, durableStoreFolder, msgPort, os.Stdout, peerSlice)
+			node, _, _, _, err := node.InitializeNode(pkString, chainOpts, useDurableStore, durableStoreFolder, msgPort, peerSlice)
```
Comment (kerzhner) [inline main.go:181]: Just checking, but it seems like this change is unrelated to the race condition. Is that correct?

**Review by kerzhner — APPROVED** [review]
Nicely done!

File: main.go (hunk 1 above)
Comment (bitwiseguy) [reply main.go:181]: Yes it is unrelated to the race condition. Just something I noticed when scanning through the code. Its related to the PR that was merged this morning
